src/ui/views/2_quantity/index.js

```diff
@@ -38,10 +38,12 @@ export class QuantityView extends React.Component {
   }
 
   subtractQuantity = () => {
-    this.setState({
+    if(this.state.quantity > this.props.params.product.minimumBuyingQuantity){
```

Comment:
yeah, this says that if its higher than, then it can be lowered, for example, if the minimum is 1, and the quantity is 2, then the quantity can be reduced, but if the quantity is 1 and the minimum is 1, the quantity can't be reduced

Reply:
You're right